docs/framework/api-architecture.md

```diff
@@ -0,0 +1,175 @@
+# API architecture
+
+
+The term 'API' refers to that code stored in the top level api folder.
```

Comment:
Is this brought over? Not quite sure what it will mean to people. I guess the point is that in CiviCRM we use the word api in a very specific sense to refer to the code in that folder. If this is just something copied over you might drop it - or get @totten to write an essay on it :-)